docker/importer/volume/surveillance-mediator/case-outcome/endpoint-composition.json

```
@@ -0,0 +1,87 @@
+{
+  "name": "FHIR - Orchestrate Case Outcome Composition Data into one resource",
+  "description": "",
+  "endpoint": {
+    "pattern": "/create-composition-case-outcome"
+  },
+  "transformation": {
+    "input": "JSON",
+    "output": "JSON"
+  },
+  "inputTransforms": {
+    "compositionId": "'urn:uuid:' & $substringAfter($string($random()), '0.')",
+    "hospitalEncounter" : "requestBody.data.encounters[0].id",
+    "caseEncounter": "requestBody.data.encounters[0].id"
+  },
+  "inputMapping": {
+    "constants.template": "composition",
+    "transforms.compositionId": "composition.fullUrl",
+    "requestBody.author": "composition.resource.author[]+",
+    "requestBody.data.identifier": "composition.resource.identifier.value",
+    "requestBody.compositionDate": "composition.resource.date",
```

@bradsawadye This date field doesn't appear to be coming through in the final Composition
```suggestion
    "requestBody.data.compositionDate": "composition.resource.date",
```

I think it's the nesting